src/components/index.ts

```diff
@@ -32,6 +32,7 @@ export * from './FrontifyPattern';
 export * from './Grid';
 export * from './InlineDialog';
 export * from './InputLabel';
+export * from './LegacyOrderableList';
```

Comment:
Did you just copy OrderableList and created the Legacy?
Are you keeping the old one as `OrderableList` for later refactoring ?

Reply:
Yup, the idea is having a new `OrderableList` using `Tree` component.
In fact i have already the new component almost ready